examples/blog/config.js

```diff
@@ -1,438 +1,434 @@
+        '<div class="panel panel-green">' +
+            '<ma-dashboard-panel collection="dashboardController.collections.recent_posts" entries="dashboardController.entries.recent_posts"></ma-dashboard-panel>' +
+        '</div>' +
+        '<div class="panel panel-green">' +
+            '<ma-dashboard-panel collection="dashboardController.collections.popular_posts" entries="dashboardController.entries.popular_posts"></ma-dashboard-panel>' +
+        '</div>' +
+    '</div>' +
+    '<div class="col-lg-6">' +
+        '<div class="panel panel-yellow">' +
+            '<ma-dashboard-panel collection="dashboardController.collections.tags" entries="dashboardController.entries.tags"></ma-dashboard-panel>' +
+        '</div>' +
+    '</div>' +
+'</div>';
+admin.dashboard(factory.dashboard()
+    .addCollection(factory.collection(post)
+        .name('recent_posts')
+        .title('Recent posts')
+        .perPage(5) // limit the panel to the 5 latest posts
+        .fields([
+            factory.field('published_at', 'date').label('Published').format('MMM d'),
+            factory.field('title').isDetailLink(true).map(truncate),
+            factory.field('views', 'number')
+        ])
+        .sortField('published_at')
+        .sortDir('DESC')
+        .order(1)
+    )
+    .addCollection(factory.collection(post)
+        .name('popular_posts')
+        .title('Popular posts')
+        .perPage(5) // limit the panel to the 5 latest posts
+        .fields([
+            factory.field('published_at', 'date').label('Published').format('MMM d'),
+            factory.field('title').isDetailLink(true).map(truncate),
+            factory.field('views', 'number')
+        ])
+        .sortField('views')
+        .sortDir('DESC')
+        .order(3)
+    )
+    .addCollection(factory.collection(comment)
+        .title('Last comments')
+        .perPage(10)
+        .fields([
+            factory.field('created_at', 'date')
+                .label('Posted'),
+            factory.field('body', 'wysiwyg')
+                .label('Comment')
+                .stripTags(true)
+                .map(truncate)
+                .isDetailLink(true),
+            factory.field('post_id', 'reference')
+                .label('Post')
+                .map(truncate)
+                .targetEntity(post)
+                .targetField(factory.field('title').map(truncate))
+        ])
+        .sortField('created_at')
+        .sortDir('DESC')
+        .order(2)
+    )
+    .addCollection(factory.collection(tag)
+        .title('Tags publication status')
+        .perPage(10)
+        .fields([
+            factory.field('name'),
+            factory.field('published', 'boolean').label('Is published ?')
+        ])
+        .listActions(['show'])
+        .order(4)
+    )
+    .template(customDashboardTemplate)
+);
+
+// add the admin configuration to angular's DI container, to make it availble from the view
+app.value('NgAdminConfiguration', admin);
```

Comment:
In order to support a fallback for the previous method (where the user doesn't define this value), you must name it differently, and keep the same meaning as before for `NgAdminConfiguration`. So the user can name the value the way she wants, then use the same name in the `<ma-ng-agmin>` configuration parameter. If there is no such parameter, the fallback is to use the global `NgAdminConfiguraiton` service, which must be defined by default.